### DISCUSSION TITLE
Software is Aborting.

### DISCUSSION THREAD

**Discussion by kumabe4r** [Raytwo/ARCropolis#328] [Issues] [answered]
![2022050104132700-1AB131B6E6571375B79964211BB3F5AE](https://user-images.githubusercontent.com/69577326/166139569-e2f8fe12-5e44-4371-a85a-b9e0f568fac2.jpg)

I know this is an issue with my mods. My problem is I don't know which ones. Is there any way to figure out what mods are causing the crash? I have latest Skyline and have tried this with ARC 3.3.0 and 3.1.0 updating from a much older version must've been from before 3.0.0 but I'm not entirely sure because I got that ARC from the creator of a modpack that i installed and have since added on to. Please let me know if there's a way to track crashes or something of the sort.

**Comment by Raytwo** [comment]
If you didn't get a message about conflicts, then your only way is removing mods bit by bit until the game runs and proceed by elimination.
We have no way of knowing which file is problematic.

**Comment by kumabe4r** [comment, marked as answer]
A bit of an update to this. It seems I was a dumb and combined my R.O.B mods into one folder which my theory is caused an issue with the conflict thing since there was a file conflict with one of them and it just made the game just wig out. Definitely a my bad on my part tho i wasn't aware this was a thing so good to know.
